assets/scss/local.scss

```diff
@@ -38,4 +38,17 @@ Removes the header from the govukNotificationBanner.
   .govuk-notification-banner__header {
     display: none;
   }
+}
+
+/**
+Custom style for the tags used on sentences that have a calculation type of recall.
+ */
+.govuk-tag--recall {
```

Comment:
Are we sure that the styling should be different to the default govuk-tag style?